formatter.go

```diff
@@ -1,9 +1,14 @@
 package logrus
 
-import "time"
+import (
+	"runtime"
+	"time"
+)
 
 const defaultTimestampFormat = time.RFC3339
 
+var EndOfLine string
```

Comment:
Do we have to make this public?

Reply:
Sorry, I will use google translate.
This is necessary to override the variable in `example_basic_test.go`. I got a build error for "go version go1.9 windows/amd64" https://ci.appveyor.com/project/sirupsen/logrus/build/46#L113 (line 113)
The magic comment "Output:" (https://golang.org/pkg/testing/#hdr-Examples) does not work with the separator "\r\n". I do not know how to fix this.